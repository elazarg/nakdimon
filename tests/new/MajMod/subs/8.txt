אֲנִי אֶקַּח אֶת הצ'קים ואפקיד אוֹתָם. בְּמָה אַתָּה תּוֹרֵם בְּדִיּוּק, לַעֲזָאזֵל? -תְּנִי לִי. אֵיפֹה הַחֲצִי הַשֵּׁנִי? הפקדתי אֶת הצ'ק שֶׁל וסט וִירְגִ'ינְיָה, וַעֲדַיִין לֹא הפקדתי אֶת הצ'ק שֶׁל קנטאקי. לָמָּה? -כִּי הָיָה שׁוֹטֵר בְּבַנְק. נוּ, אָז? -אָז? אֲנִי לֹא רוֹצֶה ששוטר ישקיף עָלַיי בִּזְמַן שֶׁאֲנִי מבצעת הוֹנָאָה. אֶת לֹא מבצעת הוֹנָאָה בקנטאקי, אֶת גָּרָה בקנטאקי. אֶת מבצעת הוֹנָאָה בווסט וִירְגִ'ינְיָה. -אַתָּה רוֹצֶה אֶת הַכֶּסֶף? קַח אוֹתוֹ. מָה אַתָּה אוֹמֵר עַל זֶה? אוּלַי אֲנִי אֶשָּׁאֵר בַּמִּיטָּה וְאַתָּה תיסע מִפֹּה לְכָל הָרוּחוֹת? וַאֲנִי לֹא רוֹצֶה לִרְאוֹת אֶת הֶחָבֵר שֶׁלְּךָ ג'וֹ בִּי מתהולל כָּאן בְּלִי לְשַׁלֵּם שָׂכָר דִּירָה. לָמָּה אַתָּה חַיָּיב לָתֵת לְכָל בֶּן זוֹנָה שהכרת בַּכֶּלֶא... אָמַרְתִּי שֶׁאֲנִי לֹא רוֹצֶה לִשְׁמוֹעַ אֶת זֶה. טוֹב? -כֵּן. יֵשׁ רַק שְׁתֵּי דְּרָכִים לְהַרְוִויחַ כֶּסֶף בָּעִיר הַזּוֹ לְאַחַר סְגִירַת המכרות: דֶּרֶךְ אַחַת הִיא עִסְקִי הַלְווֹיוֹת, והדרך הַשְּׁנִיָּיה הִיא מְכִירַת סַמִּים. לִפְעָמִים אֲנִי מִצְטַעֶרֶת שֶׁאֲנִי לֹא בְּעִסְקֵי הַלְווֹיוֹת. כָּל חַיֵּי, הַדָּבָר הַיָּחִיד שֶׁרָצִיתִי הוּא לָצֵאת מפייקוויל. לָמָּה שֶׁמִּישֶׁהוּ מִבַּחוּץ יִרְצֶה לָבוֹא לְכָאן? אֲנִי לְעוֹלָם לֹא אֵדַע.