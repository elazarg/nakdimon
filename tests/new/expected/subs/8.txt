אֲנִי אֶקַּח אֶת הַצֶּ'קִים וְאַפְקִיד אוֹתָם.
בְּמָה אַתָּה תּוֹרֵם בְּדִיּוּק, לַעֲזָאזֵל?
-תְּנִי לִי.
אֵיפֹה הַחֲצִי הַשֵּׁנִי?
הִפְקַדְתִּי אֶת הַצֶּ'ק שֶׁל וֵסְט וִירְגִ'ינְיָה,
וַעֲדַיִין לֹא הִפְקַדְתִּי אֶת הַצֶּ'ק שֶׁל קֶנְטָאקִי.
לָמָּה?
-כִּי הָיָה שׁוֹטֵר בַּבַּנְק.
נוּ, אָז?
-אָז? אֲנִי לֹא רוֹצָה שֶׁשּׁוֹטֵר יַשְׁקִיף עָלַיי בִּזְמַן שֶׁאֲנִי מְבַצַּעַת הוֹנָאָה.
אַתְּ לֹא מְבַצַּעַת הוֹנָאָה בְּקֶנְטָאקִי, אַתְּ גָּרָה בְּקֶנְטָאקִי.
אַתְּ מְבַצַּעַת הוֹנָאָה בְּוֵוסְט וִירְגִ'ינְיָה.
-אַתָּה רוֹצֶה אֶת הַכֶּסֶף? קַח אוֹתוֹ.
מָה אַתָּה אוֹמֵר עַל זֶה?
אוּלַי אֲנִי אֶשָּׁאֵר בַּמִּיטָּה וְאַתָּה תִּיסַּע מִפֹּה לְכָל הָרוּחוֹת?
וַאֲנִי לֹא רוֹצָה לִרְאוֹת אֶת הֶחָבֵר שֶׁלְּךָ גּ'וֹ בִּי מִתְהוֹלֵל כָּאן בְּלִי לְשַׁלֵּם שְׂכַר דִּירָה.
לָמָּה אַתָּה חַיָּיב לָתֵת לְכָל בֶּן זוֹנָה שֶׁהִכַּרְתָּ בַּכֶּלֶא...
אָמַרְתִּי שֶׁאֲנִי לֹא רוֹצָה לִשְׁמוֹעַ אֶת זֶה.
טוֹב? 
-כֵּן.
יֵשׁ רַק שְׁתֵּי דְּרָכִים לְהַרְוִויחַ כֶּסֶף בָּעִיר הַזּוֹ לְאַחַר סְגִירַת הַמִּכְרוֹת:
דֶּרֶךְ אַחַת הִיא עִסְקֵי הַלְּוָויוֹת, וְהַדֶּרֶךְ הַשְּׁנִיָּיה הִיא מְכִירַת סַמִּים.
לִפְעָמִים אֲנִי מִצְטַעֶרֶת שֶׁאֲנִי לֹא בְּעִסְקֵי הַלְּוָויוֹת.
כָּל חַיַּי, הַדָּבָר הַיָּחִיד שֶׁרָצִיתִי הוּא לָצֵאת מִפֶּייְקְוִויל.
לָמָּה שֶׁמִּישֶׁהוּ מִבַּחוּץ יִרְצֶה לָבוֹא לְכָאן?
אֲנִי לְעוֹלָם לֹא אֵדַע.
